src/js/components/Button/Button.js

```diff
@@ -102,7 +83,7 @@ const Button = forwardRef(
         </Box>
       );
     } else if (typeof children === 'function') {
-      contents = children({ hover, focus });
+      contents = children({ });
```

Comment:
This would be a breaking change. We would need to roll the major version of grommet to 3 to make this change. I am reluctant to roll major versions just for this.